### PR TITLE
Labs

### PR DESCRIPTION
**Pull request by Higgins00**
Please give feedback on the last 2 questions. Thank you @sierrawilde

### CODE REVIEW

**Comment by sierrawilde** [comment]
@Higgins00 
Simple statistics:
You have two 2).'s. I think that the first should be 1).
I am not sure if you really need the first plot in 2). B, because we just have to follow HW 1, but it is nice to show how the distribution looks with different lambdas
In part 3). B you need a question mark at the end and you say 'a' instead of 'an' before 'exponential'.
For part 4). it might look a bit nicer to create a table of the values, and that might make it easier for the reader to detect any patterns. I also don't really understand why you have two cells doing almost the same thing.

Non-continuous distributions:
I don't think you ran the cell under 1). B.
Adjust the widths on your bar plot so that they are touching.
You need a comma after 'trials' in your explanation under the plots.
In 1). D you spelled 'detected' wrong
I still don't really know what kind of answer that they expect from D. I had a different approach, but I don't know which is right...
1). E sounds pretty good. I think you explained it better than I did.

**Comment by Higgins00** [comment]
I have completed Lab1 and made a HTML version for it. 
@oliviatessa

File: Lab 1.html [hunk 1]
```diff
@@ -0,0 +1,12591 @@
...
+.btn-info[disabled].focus,
+fieldset[disabled] .btn-info.focus {
+  background-color: #5bc0de;
+  border-color: #46b8da;
+}
+.btn-info .badge {
+  color: #5bc0de;
+  background-color: #fff;
+}
+
```
Comment (oliviatessa) [inline Lab 1.html:11904]: What is a z-table? What are z values relationship to sigma?

Comment (oliviatessa) [inline Lab 1.html:12018]: But I don't know anything about the exponential distribution! Explain it to me!

Comment (oliviatessa) [inline Lab 1.html:12069]: Good job making these plots clearly! It's important to have legends, titles, etc.

Comment (oliviatessa) [inline Lab 1.html:12158]: I think your question could be reworded to be made more clear.

Comment (oliviatessa) [inline Lab 1.html:12275]: Explain why you'd use the sigma for a standard normal dist to describe a exp dist.

Comment (oliviatessa) [inline Lab 1.html:12394]: "This increase in sigma seems to get smaller as the value increases."

Why would that be?

**Comment by oliviatessa** [comment]
Overall, your lab could do with a lot more writing. Remember, you are trying to explain your lab as if to someone who doesn't have a statistical background. Therefore it's important to detail out what normal and exponential distributions are, what their parameters mean, etc.

**Pull request closed without merging**